docker-app/qfieldcloud/core/utils2/sentry.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import pickle
+import threading
+from io import StringIO
+from os import getenv, path
+from typing import Any, Dict
+
+from sentry_sdk import configure_scope
+
+logger = logging.getLogger(__name__)
+
+
+def report_to_sentry(name: str, request_attributes: Dict[str, Any], buffer: StringIO):
+    """On another thread, pickle the response-copy and send it to Sentry
+    or write to disk depending on the environment"""
+
+    def todo():
+        # pickling to make it easier to load to memory
+        pickled = pickle.dumps(request_attributes)
+        traceback = bytes(buffer.getvalue(), encoding="utf-8")
+
+        if getenv("ENVIRONMENT") == "development":
+            try:
+                request_path = path.abspath("/tests_artifacts/request.bin")
+                traceback_path = path.abspath("/tests_artifacts/traceback.bin")
+                with open(request_path, "wb") as fh:
+                    fh.write(pickled)
+                with open(traceback_path, "wb") as fh:
+                    fh.write(traceback)
+            except Exception as error:
+                print(
+                    f"Unable to write reports to disk at {request_path} and {traceback_path}, error: {error}"
+                )
+                print(request_attributes)
```

Comment:
I see much more `logging` than `print` used in existing code, would that be better applicable here too?

Reply:
For a reason not clear to me, logging.info doesnt write to sdout like print does under the current config. This means that the person running the test locally (as assumed by this logic branch) won't see anything.

Reply:
while `print` are convenient for developer setup, I wonder what is wrong with your setup and why `logger.info` does not work, as it is used extensively and successfully in other parts of the codebase.